src/integrationTest/java/uk/gov/hmcts/cmc/claimstore/controllers/CalculateInterestAmountTest.java

```diff
@@ -0,0 +1,42 @@
+package uk.gov.hmcts.cmc.claimstore.controllers;
+
+import org.junit.Test;
+import org.springframework.http.HttpHeaders;
+import org.springframework.test.web.servlet.MvcResult;
+import org.springframework.test.web.servlet.ResultActions;
+import uk.gov.hmcts.cmc.claimstore.BaseSaveTest;
+import uk.gov.hmcts.cmc.domain.models.InterestAmount;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.get;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
+
+public class CalculateInterestAmountTest extends BaseSaveTest {
```

Comment:
We should have a test documenting the behaviour when not all parameters required for calculation are provided. We expect some sort of 4xx error at that point.

Reply:
added

Reply:
but they are 5xx errors now as we do not map these exceptions to different status codes.